src/main/java/mate/academy/spring/model/dto/response/UserResponseDto.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.model.dto.response;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
+    private byte[] salt;
```

Comment:
As a Response, we should not send private information.

Reply:
fixed